.eslintrc.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
         "@typescript-eslint/no-unsafe-call": "off",
         "@typescript-eslint/no-unsafe-member-access": "off",
         "@typescript-eslint/no-unsafe-return": "off",
-        "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true }],
```

Comment:
This rule allows to remove properties from an object using destructuring assignment. This can be useful for a couple reasons: 
1. Write a new object without mutating the original with a terse expression, rather than cherry-picking props
1. Terse syntax for callbacks passed to iterative methods